ancar/portfolio/src/main/webapp/index.html

```diff
@@ -37,18 +43,33 @@ <h4>Some of my projects:</h4>
             <li>Flappy bird game in Racket</li>
         </ul>
       </div>
+
       <div id="Sasha">
         <p>Here is a picture of me and my cat, Sasha:</p>
         <img class="resize" src="/images/me&Sasha.jpeg"/> 
         <p id="pictures">Click <a href="gallery.html" target="_blank">here</a> to see more pictures of Sasha!</p>
+        <!--TODO : add some CSS styling for the form-->
```

Comment:
TODO(username) or TODO(#issueID) is preferred over plain TODO

I know this is only your folder, but if you get used to the practice, it will be better later, as well as we might add some linters what can enforce these at some point.